### ISSUE TITLE
How to use LOOCV protocol in TFlearn?

### ISSUE THREAD

**Issue by ruizhaogit** [tflearn/tflearn#251]
Hi TFlearn Team @aymericdamien ,

Do you have any idea about using LOOCV (leave one out cross validaiton in TFlearn)?

The protocol is like this: trian with 9 fold (such as 1~9) and validate on 1 fold (10) for 1st epoch, then  train with 9 fold (such as 2~10) and validate one 1 fold (1) for 2nd epoch. After 10 epochs average the validation accuracy. And resume training...

Is there any easy way to do this?

Best,
Rui

**Comment by aymericdamien** [comment]
It is possible, but you will have to create multiple call to `fit` function.
Example:
```python
X = X[0:9]
Y = Y[0:9]
Xtest = X[10]
Ytest = Y[10]
model.fit(X, Y)
X = X[1:8] + X[10]
Y = Y[1:8] + X[10]
Xtest = X[9]
Ytest = Y[9]
model.fit(X, Y)
```
